src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -260,13 +260,15 @@ export const deriveIsAirForceAFRBAPortal = formValues =>
   board(formValues).abbr === 'BCMR' &&
   formData(formValues).num === 149;
 
+// v2 Helpers
+
 export const answerReviewLabel = (key, formValues) => {
   const answer = formValues[key];
   const monthObj = options.months.find(
     m => String(m.value) === formValues[SHORT_NAME_MAP.DISCHARGE_MONTH],
   );
 
-  const dischargeMonth = monthObj && monthObj.label;
+  const dischargeMonth = (monthObj && monthObj.label) || '';
```

Comment:
Incidentally, I found a bug with the discharge month input where you can select a blank option from the list.

<img width="602" alt="Screenshot 2024-07-08 at 9 13 56 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/789846e5-0899-4cd1-9d3c-1cba58cb3e0a">

Reply:
Hmm I am not able to repro this locally?

Reply:
Ah, I had actually stumbled upon this in the current prod flow. Disregard; we're good here.